epitator/importers/import_geonames.py

```diff
@@ -37,13 +37,36 @@
 ]
 
 
-def read_geonames_csv():
+def windows_max_int():
+    # Find largest integer that does not cause an OverflowError for csv.field_size_limit using "binary search"
+    top = 2**32  # Largest int in Python has 64 bit but largest int in Windows has 32 bit
+    lower = 2**30  # The largest int is between 2^30 and 2^32
+    diff = top - lower
+    while diff > 1:
+        try:
+            mid = int(top - lower / 2)
```

Comment:
For this to be the midpoint between top and lower it should be `(top + lower) / 2`